src/onnx/parse_slice.cpp

```diff
@@ -97,6 +93,15 @@ struct parse_slice : op_parser<parse_slice>
         }
 
         auto ins = info.add_instruction(op, args[0]);
+        if(std::any_of(steps.begin(), steps.end(), [](auto s) { return std::abs(s) != 1; }))
+        {
+            std::vector<int64_t> nsteps;
+            std::transform(steps.begin(), steps.end(), std::back_inserter(nsteps), [](auto s) {
+                return std::abs(s);
+            });
+            ins =
+                info.add_instruction(make_op("step", {{"axes", op.axes}, {"steps", nsteps}}), ins);
```

Comment:
After checking a unit test in this PR, I think there is a problem here. Specifically,
 
If we need a reverse after the step operator, we need to do some shifts about the ends in the slice operator. 
One simple example: we have input [1, 2, 3,4], and the start is 3 and end is -INF, and we have a step 2. In this case, the output should be [4, 2]. But with our current implementation, the output is [3, 1], which is wrong.

The reason is, for our current implementation, the slice operator has the start to be 0, and end to be 4, then if we do a step of 2, we get 1, and 3, then after the reverse, we get [3, 1]. So when we have both step and reverse, we need to do some shift about the ends so the last element of the slice output is selected by the step operator. 

This issue causes the verification unit test to be incorrect.

Reply:
Makes sense. Changed the operation order in parse_slice.cpp and also related test files. Also removed the unused literals. Thanks a lot @scxiao